src/schema_formats/XML/GMP.xml.in

```diff
@@ -15950,6 +15950,13 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
         <summary>ID of filter to use to filter reports</summary>
         <type>uuid</type>
       </attrib>
+      <attrib>
+        <name>details</name>
+        <summary>
+          Whether to get the details of the reports including the
+          results, hosts, ports etc.</summary>
```

Comment:
```suggestion
          results, hosts, ports etc.
        </summary>
```

Reply:
I've updated the commit to change the formatting now.